content/07.pathogenesis.md

```diff
@@ -446,17 +445,29 @@ Data and empirically determined biological mechanisms inform models, while model
 Many current efforts to model R~t~ have also led to tools that assist the visualization of estimates in real time or over recent intervals [@url:https://epiforecasts.io/covid; @url:https://rt.live].
 These are valuable resources, yet it is also important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.
 
-### Molecular Signatures and Transmission
+### Molecular Signatures, Transmission, and Variants of Concern
 
 Genetic variation in SARS-CoV-2 has been used to elucidate patterns over time and space.
 Mutations observed in individual SARS-CoV-2 genome sequences can be used to trace transmission patterns and have provided insights during outbreak investigations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351].
 Similar mutations observed in several patients may indicate that the patients belong to the same transmission group.
 The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling future outbreaks and tracing the path of the spread of SARS-CoV-2.
-Efforts vary widely by country: the UK has coordinated has coordinated a national database of viral genomes [@doi:10/d5mg]; no such coordination has been achieved in the USA.
+Efforts vary widely by country: the UK has coordinated a national database of viral genomes [@doi:10/d5mg]; no such coordination has been achieved in the USA.
 Several studies used phylogenetic analysis to determine the source of local COVID-19 outbreaks in Connecticut (USA), [@doi:10.1016/j.cell.2020.04.021], the New York City area (USA) [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].
 There is an ongoing effort to collect SARS-CoV-2 genomes throughout the COVID-19 outbreak, and as of January 18, 2021 more than 381,000 genome sequences have been collected from patients.<!-- TODO: consider auto-updating this -->
 The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@https://www.ncbi.nlm.nih.gov/sars-cov-2/], and COVID-19 data portal [@https://www.covid19dataportal.org/].
 
+While the evolution of SARS-CoV-2 variants are discussed elsewhere within this special issue [], some of these variants may be associated with increased transmission of SARS-CoV-2. 
+For example, the coordinated sequencing efforts in the U.K. were instrumental in detecting B.1.1.7/VOC 202012/01, a variant of concern (VOC) that is associated with approximately 50% increased transmission with an estimated R0 of 1.4 due to alterations of the spike protein as a result of genetic mutations [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/955239/NERVTAG_paper_on_variant_of_concern__VOC__B.1.1.7.pdf; @doi:10.1101/2020.12.24.20248822; @doi:10.1016/j.cell.2021.01.044].
```

Comment:
```suggestion
For example, coordinated sequencing efforts of SARS-CoV-2 genomes in the U.K. were instrumental in detecting B.1.1.7/VOC 202012/01, a variant of concern (VOC) that, with an estimated R~0~ of 1.4, is associated with approximately 50% increased transmission due to genetic mutations in the _Spike_ gene that modify the Spike protein [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/955239/NERVTAG_paper_on_variant_of_concern__VOC__B.1.1.7.pdf; @doi:10.1101/2020.12.24.20248822; @doi:10.1016/j.cell.2021.01.044].<!--TO DO: What is the molecular rationale for why this is more virulent?-->
```
I think we mention the UK sequencing efforts in the evolution paper but not here, so I'm suggesting modifying the start of this sentence a bit -- but I could be mistaken about that.
Adding a comment that it would be good to try to take this to the next level pathogenesis-wise, but I don't want to hold up your PR any longer!

Reply:
I think you are right. I agree re pathogenesis, a lot of new papers out. @agitter set up an issue this week for one that we may want to consider citing here, but that we should definitely cite in the evolution paper.